content/response-to-reviewers.md

```diff
@@ -90,6 +102,10 @@ See for example https://swcarpentry.github.io/git-novice/01-basics/index.html
 > Figure .2 → This could be probably adapted as a raster plot where a dot is painted for each contribution (at the time of contribution).
 Line would be the same height and the legend would summarize the contribution.
 
+Based on this suggestion, we created an [alternative visualization](https://github.com/greenelab/meta-review/blob/6e7458dcb205a3d76daf25ee08236c24dc4de66e/analyses/deep-review-contrib/deep-review-contribution-dot.png) of Deep Review contributions that uses dots of varying sizes to show word counts by contributor on a monthly basis.
+While we thought the dot plot was a valuable supplement, there was not strong support for switching from the ridgeline plot to the dot plot for the primary manuscript visualization.
```

Comment:
I suggest following this sentence by one more that notes that our new Figure 2 caption alerts readers they can open the notebook in Binder to view the alternative visualizations or access the underlying data to make their own.